apps/dev/nextjs/pages/api/auth/[...nextauth].ts

```diff
@@ -82,7 +88,7 @@ export const authConfig: AuthConfig = {
       },
     }),
     Apple({ clientId: process.env.APPLE_ID, clientSecret: process.env.APPLE_SECRET }),
-    Auth0({ clientId: process.env.AUTH0_ID, clientSecret: process.env.AUTH0_SECRET, issuer: process.env.AUTH0_ISSUER }),
+    // Auth0({ clientId: process.env.AUTH0_ID, clientSecret: process.env.AUTH0_SECRET, issuer: process.env.AUTH0_ISSUER }),
```

Comment:
Why is Auth0 commented out here?

Reply:
thanks - i commented this out by accident when testing (was trying to run the nextjs demo app locally and didn't want to set up Auth0 env var), resolved now!